src/mixs/schema/mixs.yaml

```diff
@@ -252,31 +252,49 @@ enums:
   FaoClassEnum:
     permissible_values:
       Acrisols:
+      Alisols:
       Andosols:
+      Anthrosols:
       Arenosols:
+      Calcisols:
       Cambisols:
       Chernozems:
+      Cryosols:
+      Durisols:
       Ferralsols:
       Fluvisols:
       Gleysols:
       Greyzems:
+        deprecated: true, value no longer recognized by FAO, mixs/issues/696
```

Comment:
I would prefer that the issue link is a real github url in a see also

Reply:
I am going to push back a bit on the "I would prefer" .. there's no standard structure for this process. I would prefer the deprecated be a boolean or structured & there be a "reason" attribute so someone doesn't need to understand how see-also is a reason for deprecation. Because that's not intuitive to me. But, that's a LinkML limitation. 
So how / who gets to say when someones preference is the action to take?

I am happy to change the issue path to the ACTUAL issue URL, seems long and like it's saying the same thing to me. But, excessive clarity makes sense.